EIPS/eip-5827.md

```diff
@@ -128,69 +146,70 @@ Base method `approve(address _spender, uint256 _value)` MUST set `recoveryRate`
 
 Both `allowance()` and `transferFrom()` MUST be updated to include allowance recovery logic.
 
-`_value` within `approveRenewable(address _spender, uint256 _value, uint256 _recoveryRate)` method sets both initial allowance amount and the maximum allowance limit it can regain to.
+`approveRenewable(address _spender, uint256 _value, uint256 _recoveryRate)` MUST set both the initial allowance amount and the maximum allowance limit to which it can recover to `value`.
```

Comment:
```suggestion
`approveRenewable(address _spender, uint256 _value, uint256 _recoveryRate)` MUST set both the initial allowance amount and the maximum allowance limit to `_value`.
```

do you think it still makes sense if we drop the "to which it can recover" bit? it's a little bit awkward imo

Reply:
```suggestion
`approveRenewable(address _spender, uint256 _value, uint256 _recoveryRate)` MUST set both the initial allowance amount and the maximum allowance limit (which the allowance can recover to) to `value`.
```